subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/verification/verifier/DependencyVerifierBuilder.java

```diff
@@ -169,9 +172,10 @@ void addIgnoredKey(ModuleComponentArtifactIdentifier artifact, IgnoredKey key) {
             byArtifact.computeIfAbsent(artifact.getFileName(), id -> new ArtifactVerificationBuilder()).addIgnoredKey(key);
         }
 
-        private static ArtifactVerificationMetadata toArtifactVerification(Map.Entry<String, ArtifactVerificationBuilder> entry) {
+        private static ArtifactVerificationMetadata toArtifactVerification(Map.Entry<String, ArtifactVerificationBuilder> entry) throws InvalidGpgKeyIds {
```

Comment:
Even if not necessary, I've made it obvious for the next guy that this can throw `InvalidGpgKeyIds`.